### PR TITLE
 BaSP-MR-19: Tests for admins, POST and DELETE methods

### PR DESCRIPTION
**Pull request by fernandofernandezuy**
I worked on creating a test suite using Jest to verify the correct behavior of the API endpoints for creating and deleting admin. It ensures that the endpoints return the expected status codes and that the database its modified correctly.

### CODE REVIEW

File: src/tests/admins.test.js [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
+      .post('/api/admins')
+      .send(mockAdmin);
+    expect(response.statusCode).toBe(201);
+  });
+  test('POST: Admin should have a valid password', async () => {
+    const response = await request(app)
+      .post('/api/admins')
+      .send(mockAdmin);
+    const admin = response.body.data;
+    expect(admin.password).toMatch(/^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/);
```
Comment (GaloDurante) [inline src/tests/admins.test.js:33]: You shouldn't verify the password here. Instead verify it on validations.

File: src/tests/admins.test.js [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+    let response = await request(app)
+      .post('/api/admins')
+      .send(mockAdmin);
+    // eslint-disable-next-line no-underscore-dangle
+    const newAdminId = response.body.data._id;
+    response = await request(app)
+      .get(`/api/admins/${newAdminId}`);
+    expect(response.statusCode).toBe(200);
+  });
+});
```
Comment (FontanR) [inline src/tests/admins.test.js:45]: you could check every item with 1 post

**Review by FontanR — APPROVED** [review]

**Review by Eugeniagassman — APPROVED** [review]
good for me

**Review by leo-balonchard — APPROVED** [review]
Looks good

**Pull request merged**